### PR TITLE
Add peak prominence option

### PR DESCRIPTION
**Pull request by davidfee5**
This PR adds the option to include the peak prominence when finding maxima in the stack. There are other peak parameters we could include but prominence seems like the next most useful.

### CODE REVIEW

File: rtm/stack.py [hunk 1]
```diff
@@ -86,19 +89,20 @@ def get_peak_coordinates(S, global_max=True, height=None, min_time=None,
 
     else:
 
-        if (height is None) or (min_time is None):
-            raise ValueError('height and min_time must be supplied for peak '
-                             'detection when global_max=False!')
+        if (height is None) or (min_time is None) or (prominence is None):
```
[the remark this answers is not in the document]
Comment (davidfee5) [reply rtm/stack.py:92]: Sure, that makes sense. I'll try to modify accordingly.

File: rtm/stack.py [hunk 2]
```diff
@@ -94,11 +97,12 @@ def get_peak_coordinates(S, global_max=True, height=None, min_time=None,
...
         # Find all peaks based on set thresholds
-        peaks, props = find_peaks(s_peak, height, distance=min_time/peak_dt)
+        peaks, props = find_peaks(s_peak, height, distance=min_time/peak_dt,
+                                  prominence=prominence)
 
         npeaks = len(peaks)
-        print(f'Found {npeaks} peaks in stack for height > {height:.1f} and '
-              f'min_time > {min_time:.1f} s.')
+        print(f'Found {npeaks} peaks in stack for height > {height:.1f}, '
+              f'min_time > {min_time:.1f} s, and prominence > {prominence}.')
```
Comment (davidfee5) [inline rtm/stack.py:105]: Should this be printed if `prominence=None`? I think maybe no.

Comment (liamtoney) [reply rtm/stack.py:105]: I agree that it should not be printed.

Comment (davidfee5) [reply rtm/stack.py]: OK updated @liamtoney. Let me know if you think ready.

**Review by liamtoney — APPROVED** [review]

**Pull request merged**